index.html

```diff
@@ -44,7 +44,7 @@
           svg.selectAll("path")
             .data(layer_data.features.sort(function(a, b) { return a.properties.sort_key ? a.properties.sort_key - b.properties.sort_key : 0 }))
           .enter().append("path")
-            .attr("class", function(d) { var kind = d.properties.kind || ''; if(d.properties.boundary=='yes'){kind += '_boundary';} return layer + ' ' + kind; })
+            .attr("class", function(d) { var kind = d.properties.kind || ''; if(d.properties.boundary=='yes'){kind += '_boundary';} if(d.properties.label_position=='yes'){kind += '_label_position';} return layer + '-layer ' + kind; })
```

Comment:
It should be called `_label_placement` to match what's in the tiles (thanks for the catch, @rmarianski). 

It's a cheap way of marking points instead of lines or polygons useful for future debugging (not used in styling now).